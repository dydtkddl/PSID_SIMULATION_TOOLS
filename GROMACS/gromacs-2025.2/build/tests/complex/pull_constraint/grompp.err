                      :-) GROMACS - gmx grompp, 2025.2 (-:

Executable:   /home/yongsang/Research/PSID_server_room/GROMACS/gromacs-2025.2/build/bin/gmx
Data prefix:  /home/yongsang/Research/PSID_server_room/GROMACS/gromacs-2025.2 (source tree)
Working dir:  /home/yongsang/Research/PSID_server_room/GROMACS/gromacs-2025.2/build/tests/complex/pull_constraint
Command line:
  gmx grompp -f /home/yongsang/Research/PSID_server_room/GROMACS/gromacs-2025.2/build/tests/regressiontests-2025.2/complex/pull_constraint/grompp.mdp -c /home/yongsang/Research/PSID_server_room/GROMACS/gromacs-2025.2/build/tests/regressiontests-2025.2/complex/pull_constraint/conf -r /home/yongsang/Research/PSID_server_room/GROMACS/gromacs-2025.2/build/tests/regressiontests-2025.2/complex/pull_constraint/conf -p /home/yongsang/Research/PSID_server_room/GROMACS/gromacs-2025.2/build/tests/regressiontests-2025.2/complex/pull_constraint/topol -maxwarn 10 -n /home/yongsang/Research/PSID_server_room/GROMACS/gromacs-2025.2/build/tests/regressiontests-2025.2/complex/pull_constraint/index

Ignoring obsolete mdp entry 'optimize_fft'
Ignoring obsolete mdp entry 'ns_type'
Replacing old mdp entry 'nstxtcout' by 'nstxout-compressed'
Replacing old mdp entry 'xtc_precision' by 'compressed-x-precision'

NOTE 1 [file /home/yongsang/Research/PSID_server_room/GROMACS/gromacs-2025.2/build/tests/regressiontests-2025.2/complex/pull_constraint/grompp.mdp]:
  You have set rlist larger than the interaction cut-off, but you also have
  verlet-buffer-tolerance > 0. Will set rlist using
  verlet-buffer-tolerance.

Generating 1-4 interactions: fudge = 0.5
Pull group 1 'r_1' has 3 atoms
Pull group 2 'r_2' has 3 atoms
Pull group 3 'r_3' has 3 atoms
Number of degrees of freedom in T-Coupling group System is 1292.00
Pull group  natoms  pbc atom  distance at start  reference at t=0
       1         3         2
       2         3         5       0.500 nm          0.500 nm
       1         3         2
       3         3         8       0.726 nm          0.400 nm

There was 1 NOTE

Thanx for Using GROMACS - Have a Nice Day

