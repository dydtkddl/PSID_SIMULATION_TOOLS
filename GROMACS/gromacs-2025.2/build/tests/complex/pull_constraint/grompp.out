Generated 330891 of the 330891 non-bonded parameter combinations

Generated 330891 of the 330891 1-4 parameter combinations

Excluding 2 bonded neighbours molecule type 'SOL'

Taking velocities from '/home/yongsang/Research/PSID_server_room/GROMACS/gromacs-2025.2/build/tests/regressiontests-2025.2/complex/pull_constraint/conf.gro'

Determining Verlet buffer for a tolerance of 0.005 kJ/mol/ps at 298.15 K

Calculated rlist for 1x1 atom pair-list as 0.896 nm, buffer size 0.096 nm

Set rlist, assuming 4x4 atom pair-list, to 0.871 nm, buffer size 0.071 nm

Note that mdrun will redetermine rlist based on the actual pair-list setup

This run will generate roughly 0 Mb of data
